<!DOCTYPE html>

<html lang="en">

<head>
  <!-- http://getbootstrap.com/docs/4.5/ -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
    <script src="https://app.awesome-table.com/AwesomeTableInclude.js"></script>
    <link rel='stylesheet' href='/static/styles.css'>
    <link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
    <script src="myscripts.js"></script>
    <title>AlleleFinder: About</title>
</head>
<body>
    <div id='header'>
        <h1>AlleleFinder</h1>
        <nav class="navbar navbar-expand-md navbar-light bg-light border header">
            <div class="collapse navbar-collapse" id="navbar">
                <ul class="navbar-nav mr-auto mt-2">
                    <li class="nav-item"><a class="nav-link" href="/">Home</a></li>
                    <li class="nav-item"><a class="nav-link" href="/consolidated">Consolidated Data Format</a></li>
                    <li class="nav-item"><a class="nav-link" href="/about">About</a></li>
                </ul>
            </div>
        </nav>
    </div>
    <main class="container p-5">
        <article>
            <h2>
                alleleFinder: Germline Allele-Frequencies Aligned to Single Nucleotide Somatic Gene Mutations by Ancestry Groups
            </h2>
            <p>
                Rebecca D. Pollard<sup>*1,2</sup>, Matthew D. Wilkerson<sup>3,4</sup>
            </p>
            <p class='sub'>1. Maret School, 3000 Cathedral Ave NW, Washington, DC 20008</p>
            <p class='sub'>2. Metis Foundation, 300 Convent St #1330, San Antonio, TX 78205, USA</p>
            <p class='sub'>3. The American Genome Center, Precision Medicine Initiative for Military Medical Education and Research (PRIMER), Uniformed Services University of the Health Sciences, Bethesda, MD, 20814, USA</p>
            <p class='sub'>4. Department of Anatomy, Physiology and Genetics, Uniformed Services University of the Health Sciences, Bethesda, MD, 20814, USA</p>
            <h3>
                Abstract
            </h3>
            <p>
                Existing medical genetics databases predominantly comprise sequences representing European individuals, potentially leading to biases in genetic research and clinical practice. The issue arises from the misclassification of normal population variants that are not covered by the standard European reference sequences. These variants can be misidentified as disease-causing somatic mutations, despite being normal germline variants specific to a particular ancestral group. To address this limitation, we developed alleleFinder, a comprehensive repository providing access to germline allele frequencies associated with somatic gene mutations across multiple ancestries. The ancestries included are African, Latino/Admixed American, Ashkenazi Jewish, East Asian, Finnish European, Estonian, North-Western European, and Southern European. To build alleleFinder, Germline allele frequencies were extracted from the Catalogue of Somatic Mutations in Cancer v96 (COSMIC) and Genome Aggregation Database (GnomAD) using Linkage Disequilibrium V2. Data disparities were identified and addressed, illuminating deficiencies in current datasets. AlleleFinder facilitates the interpretation of somatic DNA sequencing by allowing investigators to ascertain the statistical likelihood of misidentifying a germline-gene mutation as somatic across different ancestries. By enabling access to a broader range of population-based genomic variant annotations, this tool enhances the interpretation of genomic sequencing data and helps to mitigate potential disparities.
            </p>

        </article>
    </main>
</body>
<footer>
    <span>Contact: bpollard@maret.org</span>
    <br>
    <span>Data from </span>
    <a href="https://gnomad.broadinstitute.org/">GnomAD</a>
    <span> and </span>
    <a href="https://cancer.sanger.ac.uk/cosmic">COSMIC</a>
</footer>
</html>

